Missing peer dependency for mui-material-nextjs

### Steps to reproduce

Steps:
1. Create next js app with mui
2. Convert to yarn berry with pnp
3. Run the app

### Current behavior

Receive the following error:

```
Import trace for requested module:
../../.yarn/__virtual__/@mui-material-nextjs-virtual-3a287004dc/3/.yarn/berry/cache/@mui-material-nextjs-npm-5.15.11-c1c7f34dd3-10c0.zip/node_modules/@mui/material-nextjs/v13-appRouter/index.js
../../.yarn/__virtual__/@mui-material-nextjs-virtual-3a287004dc/3/.yarn/berry/cache/@mui-material-nextjs-npm-5.15.11-c1c7f34dd3-10c0.zip/node_modules/@mui/material-nextjs/v14-appRouter/index.js
./src/contexts/theme.tsx
 ⨯ ../../.yarn/__virtual__/@mui-material-nextjs-virtual-3a287004dc/3/.yarn/berry/cache/@mui-material-nextjs-npm-5.15.11-c1c7f34dd3-10c0.zip/node_modules/@mui/material-nextjs/v13-appRouter/appRouterV13.js:6:1
Module not found: Can't resolve '@emotion/react'
```

### Expected behavior

It should work seemlessly

### Context

_No response_

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  System:
    OS: macOS 14.5
  Binaries:
    Node: 20.15.0 - ~/.nvm/versions/node/v20.15.0/bin/node
    npm: 10.7.0 - ~/.nvm/versions/node/v20.15.0/bin/npm
    pnpm: Not Found
  Browsers:
    Chrome: 126.0.6478.127
    Edge: Not Found
    Safari: 17.5
```
</details>


**Search keywords**: yarn berry pnp peer dependency

--- a/packages/mui-material-nextjs/package.json
+++ b/packages/mui-material-nextjs/package.json
@@ -48,6 +48,7 @@
   },
   "peerDependencies": {
     "@emotion/cache": "^11.11.0",
+    "@emotion/react": "^11.11.4",
     "@emotion/server": "^11.11.0",
     "@mui/material": "workspace:*",
     "@types/react": "^17.0.0 || ^18.0.0",

